test/many_ode.jl

```diff
@@ -8,27 +8,27 @@ const _order = 28
 const _abstol = 1.0E-20
 const tT = Taylor1(_order)
 
-@testset "Tests: dot{x}=x.^2, x(0) = [3.0,1.0]" begin
-    function eqs_mov!(t, x, Dx)
+@testset "Tests: dot{x}=x.^2, x(0) = [3, 1]" begin
+    function eqs_mov!(Dx, x, p, t)
         for i in eachindex(x)
             Dx[i] = x[i]^2
         end
         nothing
     end
     exactsol(t, x0) = x0/(1.0-x0*t)
     t0 = 0.0
-    q0 = [3.0, 1.0]
-    q0T = [Taylor1(q0[1], _order), Taylor1(q0[2], _order)]
+    q0 = [3, 1]
+    q0T = [Taylor1(3.0, _order), Taylor1(1.0, _order)]
```

Comment:
Maybe this could be simplified to `q0T = Taylor1.(q0, _order)`

Reply:
Good point. I'll change it.

Reply:
Mmmm. Rereading the code I recall that the change has a purpose. `q0T` is passed to `jetcoeffs!`, so it should be a `Taylor1{Float64}` vector. I'll thus change it to `Taylor1.(Float64.(q0), _order)`